src/combine.cpp

```diff
@@ -1174,6 +1174,16 @@ void do_symbol_check(chunk_t *prev, chunk_t *pc, chunk_t *next)
                   }
                }
             }
+            // Issue #322 STDMETHOD(GetValues)(BSTR bsName, REFDATA** pData);
+            if (pc->next->next != NULL)
+            {
+               if (pc->next->next->type == CT_STAR)
```

Comment:
Yes, that's what I meant. Right now it looks like:
`if(a) { if (b) { ... } }`
and it could simply be:
`if(a && b) { ... }`